admin/src/scenes/dashboardV2/visitor/subscenes/inscription/index.jsx

```diff
@@ -1,23 +1,147 @@
-import React from "react";
-import ButtonPrimary from "../../../../../components/ui/buttons/ButtonPrimary";
-import DashboardContainer from "../../../components/DashboardContainer";
+import queryString from "query-string";
+import React, { useEffect, useMemo, useState } from "react";
+import { HorizontalBar } from "../../../components/graphs";
+
+import { useSelector } from "react-redux";
+import { toastr } from "react-redux-toastr";
+import { COHORTS, departmentToAcademy, REFERENT_ROLES, region2department } from "snu-lib";
+import api from "../../../../../services/api";
+import { FilterDashBoard } from "../../../components/FilterDashBoard";
+import StatutPhase from "../../../components/inscription/StatutPhase";
+
+import { orderCohort } from "../../../../../components/filters-system-v2/components/filters/utils";
+import plausibleEvent from "../../../../../services/plausible";
+import { getNewLink } from "../../../../../utils";
+import { getFilteredDepartment } from "../../../components/common";
+import Details from "../../../components/inscription/Details";
+import TabSchool from "../../../components/inscription/TabSchool";
 
 export default function Index() {
+  const user = useSelector((state) => state.Auth.user);
+  const [inscriptionGoals, setInscriptionGoals] = useState();
+
+  const [inscriptionDetailObject, setInscriptionDetailObject] = useState({});
+
+  const [filterArray, setFilterArray] = useState([]);
+  const [departmentOptions, setDepartmentOptions] = useState([]);
+  const academyOptions = [...new Set(region2department[user.region].map((d) => departmentToAcademy[d]))].map((a) => ({ key: a, label: a }));
+
+  useEffect(() => {
+    let filters = [
+      {
+        id: "academy",
+        name: "Académie",
+        fullValue: "Toutes",
+        options: academyOptions.sort((a, b) => a.label.localeCompare(b.label)),
+      },
+      {
+        id: "department",
+        name: "Département",
+        fullValue: "Tous",
+        options: departmentOptions,
+      },
+      {
+        id: "cohort",
+        name: "Cohorte",
+        fullValue: "Toutes",
+        options: COHORTS.map((cohort) => ({ key: cohort, label: cohort })),
+        sort: (e) => orderCohort(e),
+      },
+    ].filter((e) => e);
+    setFilterArray(filters);
+  }, [departmentOptions]);
+
+  const [selectedFilters, setSelectedFilters] = React.useState({
+    cohort: ["Février 2023 - C", "Avril 2023 - A", "Avril 2023 - B", "Juin 2023", "Juillet 2023", "Octobre 2023 - NC"],
+  });
```

Comment:
Je me demande si on a pas une constante dans `snu-lib`. (Ping @hlecourt)

```suggestion
  const [selectedFilters, setSelectedFilters] = useState({
    cohort: ["Février 2023 - C", "Avril 2023 - A", "Avril 2023 - B", "Juin 2023", "Juillet 2023", "Octobre 2023 - NC"],
  });
```